gpMgmt/bin/gppylib/commands/base.py

```diff
@@ -452,7 +454,7 @@ def execute(self, cmd, wait=True, pickled=False):
         # executable='/bin/bash' is to ensure the shell is bash.  bash isn't the
```

Comment:
Nit: This comment is outdated.

```suggestion
exectable=findCmdInPath("bash") is to ensure the shell is bash.  bash isn't the
```